src/components/DemoVideo.tsx

```diff
@@ -10,12 +10,12 @@ const DemoVideo = ({ visible }: { visible: boolean }) => {
       className="w-full h-full"
       width="100%"
       height="100%"
-      frameBorder="0"
       allowFullScreen
+      frameBorder="0"
+      src="https://www.youtube.com/embed/8Cw5ktNADBQ"
```

Comment:
we need autoplay to be enabled for this (Andrew requested it to avoid double clicking play) 

For JSX you also do not need to close an element but assuming it was your linter.

<iframe /> is the same as <iframe></iframe> (if there are no children)

Reply:
cool. will put the autoplay back